test/k6/src/config.js

```diff
@@ -44,6 +44,9 @@ export var notifications = {
   },
   notifications_sms: function (orderId) {
     return `https://platform.${baseUrl}/notifications/api/v1/orders/${orderId}/notifications/sms/`;
+  },
+  conditionCheck: function(conditionMet) {
+    return `http://altinn-notifications.default.svc.cluster.local/notifications/api/v1/tests/sendcondition?conditionMet=${conditionMet}`;
```

Comment:
this would require changes if we ever move the deployment out of the default namespace in AKS. Would recommend that we rather use the endpoint exposed via APIM so we don't have a hidden dependency here

Reply:
How about we make the full url an input param in the test?

Reply:
We can also generate a url based on environment like we do with other urls.

Reply:
Adding subscription key as a separate value for now. With some dynamic changes to the url to change the condition result I find it best to generate the url in code.